MatrixKit/Views/MXKAlert.m

```diff
@@ -78,13 +78,14 @@ - (NSInteger)addActionWithTitle:(NSString *)title style:(MXKAlertActionStyle)sty
     {
         index = [(UIAlertController *)_alert actions].count;
         
+        __weak typeof(self) weakSelf = self;
         UIAlertAction* action = [UIAlertAction actionWithTitle:title
                                                          style:(UIAlertActionStyle)style
                                                        handler:^(UIAlertAction * action) {
                                                            
                                                            if (handler)
                                                            {
-                                                               handler(self);
+                                                               handler(weakSelf);
```

Comment:
After thinking, I agree with you we can't call any action's handler block since MXAlert is released. This sanity check is not relevant.